src/pages/Details.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import HelpLine from '../components/HelpLine';
+import MovieList from '../components/MovieList';
+
+import mockObject from '../MockFilmList';
+
+export default class Details extends React.PureComponent {
+    constructor(props) {
+        super(props);
+
+        this.updateCurrentMovie();
+    }
+
+    updateCurrentMovie () {
+        this.currentMovie = mockObject.find((el) => (
+            String(el.id) === this.props.match.params.movieID
+        ));
+    }
+
+    render() {
+        return (
+            <React.Fragment>
+                <React.Fragment>
```

Comment:
Looks like one of this <React.Fragment> is redundant

Reply:
Done